freeipa/src/main/java/com/sequenceiq/freeipa/converter/backup/BackupConverter.java

```diff
@@ -53,6 +54,11 @@ private void decorateBackupFromLoggingRequest(Backup backup, LoggingRequest logg
                 adlsGen2Params.setSecure(adlsGen2FromRequest.isSecure());
                 adlsGen2Params.setManagedIdentity(adlsGen2FromRequest.getManagedIdentity());
                 backup.setAdlsGen2(adlsGen2Params);
+            } else if (loggingRequest.getGcs() != null) {
+                GcsCloudStorageV1Parameters gcsParams = new GcsCloudStorageV1Parameters();
+                GcsCloudStorageV1Parameters gcsFromRequest = loggingRequest.getGcs();
+                gcsParams.setServiceAccountEmail(gcsFromRequest.getServiceAccountEmail());
+                backup.setGcs(gcsFromRequest);
```

Comment:
I think the argument should be `gcsParams` instead to be consistent with S3 / ADLSg2 above.